src/main/java/org/opentripplanner/model/trafficprediction/SpeedData.java

```diff
@@ -0,0 +1,41 @@
+package org.opentripplanner.model.trafficprediction;
+
+public class SpeedData {
+
+    private int id;
+    private long clusterId;
+    private int currentSpeed;
+    private String measureTime;
```

Comment:
Co jest w tym `measureTime`? Jak to się ma do uzgodnionego formatu? https://appnroll.atlassian.net/wiki/spaces/planner/pages/415170898/Traffic+prediction+format

Reply:
natazie wjsonie prszesyjla czyste dane  zbazy bo  pylo troche kotrowersji na temat tego jak generowac konkretne przedzialy obowiazywania danego przedkoscu w danym mommecie ostattetecznie rozmawialem diaj otym dosc dlugo zpawlem i doszlimy do wnisku ze dala dane chwili czasu jako predykcje bedziemy zwracac  sredbua wzoba  po odlehlosci od dwoch koejnych punktow pomiaru  wiec tego sie nie da pszeslac w tatmtm naszym jsosonwy formacie wiec pozwolillem   musialem niestety tutaj odniego ostapic bo inaczej caly czas bym wisial na ustalleniu ostatenczne metody predykcji aby wygenerowac format docelowy jjsona pytanie jak bardzo chemy do niego wrocic vo przesylanie do planera surowych danych na swoje wady i zalety
jesli nie chcemy trzymac wszystki pomiarow wplanerze to oczywiscie  m mpzna napodstawie tej klasy wyhenerowac juz klase z docelowymi przedzialdzialami moge to nnapisac ale pszy iloscui pomiarow jakie mozemy wykonac to poki co nie powinien byc problem

Reply:
w takim razie może na razie zostać tak, jak napisałeś